ecl/eclagent/eclagent.cpp

```diff
@@ -2942,13 +2941,8 @@ char * EclAgent::getGroupName()
                     if (strieq(thorName, envClusters.item(j)))
                     {
                         const char *envGroup = envGroups.item(j);
-                        if (groupName.length())
-                        {
-                            if (!strieq(groupName, envGroup))
-                                throw MakeStringException(-1, "getGroupName(): ambiguous groups %s, %s", groupName.str(), envGroup);
-                        }
-                        else
-                            groupName.append(envGroup);
+                        if (!groupNames.contains(envGroup))
```

Comment:
previously it was a stricmp() between goupName and envGroup.  Is contains case-insensitive ?  Does it matter ?